tools/validate/src/builds/index.ts

```diff
@@ -31,15 +30,32 @@ export const validateBuilds = ({
 
     // Check that every package's /dist folder has a valid UMD, ESM & TS files
     for (const pkgPath of packagePaths) {
-      const pkgName = getPackageName(pkgPath);
+      const pkgJson = getPackageJson(pkgPath);
 
-      if (!pkgName) {
+      if (!pkgJson) {
         exit1('Invalid package path: ' + pkgPath);
         return;
       }
 
-      // Skip packages
+      // extract the name and output paths from the package.json
+      const {
+        name: pkgName,
+        main: umdPath,
+        module: esmPath,
+        types: typesPath,
+      } = pkgJson;
+
+      const checks = {
```

Comment:
Are these defaults?

Reply:
we default to assuming everything passes, and then set it to false if it should exist but doesn't (L70-93)